src/processbuildevents.py

```diff
@@ -34,4 +34,7 @@ def handler(build_event, context):
     if config.DELETE_PREVIOUS_COMMENTS:
         GITHUB.delete_previous_comments(build)
 
-    GITHUB.publish_pr_comment(build)
+    if build.status == 'SUCCEEDED' and not config.COMMENT_ON_SUCCESS:
+        LOG.info('Not publishing comment because build SUCCEEDED but COMMENT_ON_SUCCESS is set to False.')
```

Comment:
I try to keep info logging very minimal since CW Logs costs can add up. Users can always flip the log level to DEBUG if the app doesn't seem to be working as expected.

```suggestion
        LOG.debug('Not publishing comment because build SUCCEEDED but COMMENT_ON_SUCCESS is set to false.')
```